app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class QuestionsController < ApplicationController
```

Comment:
I don't think this ever gets used correct? Since questions are created and viewed through the form nothing here is ever called I believe. If so this can be deleted.